syslibs/comm/kshttp/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ CMAKE_MINIMUM_REQUIRED(VERSION 3.11)
 
 SET(LIB_NAME kshttp)
 
-FIND_PACKAGE(TCL REQUIRED)
+FIND_PACKAGE(TCLSH REQUIRED)
```

Comment:
Opted to just swap Macros so that the additional logic already implemented in CMake Macros is used

Did not add additional check for TCLSH_FOUND because build would automatically fail because package is marked as required